### PR TITLE
docs(buildroot): add guide to build and embed lvgl on br image

### PR DESCRIPTION
**Pull request by EDGEMTech-GabrielC**
Added Buildroot documentation: 
- Start with Buildroot
- Create an image with Buildroot
- Compile LVGL application with the image SDK and embed it on the image.

This guide provides a detailed, step-by-step process for setting up a clean project using Buildroot. The "Quick Start" section outlines the essential commands to quickly get results. It demonstrates how to configure an image for the Raspberry Pi 4, including embedding the LVGL benchmark demo. The same approach applies to other boards, as the concepts remain consistent. For further customization options, please refer to the Buildroot user manual.

### CODE REVIEW

File: docs/integration/os/buildroot/quick_setup.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+A Git repository is available that includes everything needed to test the 
+Buildroot setup without following the guide. It is intended for testing 
+purposes and can be used if you are already familiar with Buildroot concepts 
+and architecture.
+
+Get the repository
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/EDGEMTech/lvgl_buildroot_example.git
```
Comment (kisvegabor) [inline docs/integration/os/buildroot/quick_setup.rst]: It should be hosted in the LVGL organization. We can create an `lv_buildroot` repo.

**Review by kisvegabor — COMMENTED** [review]
Thank you!

File: docs/integration/os/buildroot/quick_setup.rst (hunk 1 above)
Comment (EDGEMTech-GabrielC) [reply docs/integration/os/buildroot/quick_setup.rst]: I agree, way better to have it in LVGL organization.

**Review by kisvegabor — DISMISSED** [review]
I haven't tested it but the type of information is exactly what I imagined.

**Comment by kisvegabor** [comment]
@EDGEMTech-GabrielC we have merged a larger docs reorganization PR. Could you fix the merge conflicts. After I'll approve it. 

Sorry for the trouble.

**Review by kisvegabor — APPROVED** [review]
Thank you!

File: docs/details/integration/os/buildroot/quick_setup.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+A Git repository is available that includes everything needed to test the 
+Buildroot setup without following the guide. It is intended for testing 
+purposes and can be used if you are already familiar with Buildroot concepts 
+and architecture.
+
+Get the repository
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/lvgl/lv_buildroot.git
```
Comment (XuNeo) [inline docs/details/integration/os/buildroot/quick_setup.rst:13]: Could you please add the link to this doc in lv_buildroot repo's README.md?

**Review by XuNeo — APPROVED** [review]
The documentation is very detailed—great job!

**Review by FASTSHIFT — APPROVED** [review]

File: docs/details/integration/os/buildroot/quick_setup.rst (hunk 2 above)
Comment (EDGEMTech-GabrielC) [reply docs/details/integration/os/buildroot/quick_setup.rst:13]: Glad to hear that! 
I added a PR on lv_buildroot repository.
https://github.com/lvgl/lv_buildroot/pull/1

**Pull request merged**